app/config/config.neon

```diff
@@ -51,6 +51,10 @@ parameters:
   monitor:  # connection to monitor
     address: wss://recodex.mff.cuni.cz:4443/ws  # URI of monitor
 
+  registration:
+    enabled: false  # for security reasons, simple registration is disabled by default
```

Comment:
This option should definitely be present in `config.local.neon.example`, so it will be visible to new-coming admins.